docs/VictoriaLogs/FAQ.md

```diff
@@ -175,3 +175,21 @@ And for the following log, its `_msg` will be `foo bar in body`:
   "body": "foo bar in body"
 }
 ```
+
+## What is the max size of a log record?
+
+VictoriaLogs is optimized for logs with records of rather small length.
+Specifically, the total length of of all fields in a log record shouldn't
+exceed a few kilobytes. The max length the VictoriaLogs is capable of
+handling efficiently is `2MiB`. This limit is hadrcoded and is unlikely to
+change.
+
+Why 2MiB? VictoriaLogs stores log data in blocks which are processed
+(compressed during ingestion and decompressed during retrieval) independently.
+While bigger blocks are compressed more effectively, they result in greater
+search costs because in order to find a single record an entire block needs to
+be decompressed. So the size should be a compromise between the compression rate
+and search costs. The optimal size seems to be 2MiB. No benchmarks have been
```

Comment:
I wouldn't mention missing benchmarks here, since it raises additional questions on why these benchmarks weren't conducted and which optimal block size limit should be used.

The purpose of this FAQ entry is to answer the question on the maximum size of log record supported by VictoriaLogs, in a clear and concise way.

It is enough mentioning the fact that blocks are processed in an atomic way (the whole block at once) and 2MB fits L2 cache for typical CPUs, so this gives an optimal performance when processing such blocks.